bool status
float32 current
float32 linear_speed